apps/api/v2/src/modules/teams/schedules/services/teams-schedules.service.ts

```diff
@@ -0,0 +1,41 @@
+import { SchedulesRepository_2024_06_11 } from "@/ee/schedules/schedules_2024_06_11/schedules.repository";
+import { OutputSchedulesService_2024_06_11 } from "@/ee/schedules/schedules_2024_06_11/services/output-schedules.service";
+import { TeamsRepository } from "@/modules/teams/teams/teams.repository";
+import { Injectable } from "@nestjs/common";
+
+import { ScheduleOutput_2024_06_11 } from "@calcom/platform-types";
+
+@Injectable()
+export class TeamsSchedulesService {
+  constructor(
+    private readonly teamsRepository: TeamsRepository,
+    private readonly schedulesRepository: SchedulesRepository_2024_06_11,
+    private readonly outputSchedulesService: OutputSchedulesService_2024_06_11
+  ) {}
+
+  async getTeamSchedulesByUserIds(teamId: number, skip = 0, take = 250) {
```

Comment:
Why do we have `getTeamSchedulesByUserIds` and `getTeamSchedules` - their function body looks the same.

I assume we can get rid of `getTeamSchedulesByUserIds` (it also is named by user ids but does not take any user ids as a parameter so has an inaccurate name).